packages/datagrid/src/graphicscontext.ts

```diff
@@ -415,7 +415,7 @@ class GraphicsContext implements IDisposable {
     return this._context.createRadialGradient(x0, y0, r0, x1, y1, r1);
   }
 
-  createPattern(image: HTMLImageElement | HTMLCanvasElement | HTMLVideoElement, repetition: string): CanvasPattern {
+  createPattern(image: HTMLImageElement | HTMLCanvasElement | HTMLVideoElement, repetition: string): CanvasPattern | null {
```

Comment:
This is the only line I can think of that might break something downstream, but this should really be considered a "bug fix" (fixing an incorrect typing).

Reply:
I Agree.